src/peerslist.test.js

```diff
@@ -0,0 +1,44 @@
+const {getSwarms} = require('./peerslist');
+
+const { spawn } = require('child_process');
+
+const http = require('http');
+
+const fs = require('fs');
+
+const assert = require('assert');
+
+
+const sample_JSON = '{ "a": 123 }';
+
+describe('peerslist tests', () => {
+
+
+    let server;
+
+    before(() => {
+
+        server = http.createServer((req, res) => {
+
+            res.writeHead(200, {'Content-Type': 'text/plain'});
+            res.end(sample_JSON);
+
+        }).listen(8080);
```

Comment:
This test will fail if you have any other 8080 service on your machine, please use a more robust approach.

Reply:
Added user configurable port from ENV in new commit.